classes/item.lua

```diff
@@ -196,13 +208,27 @@ end
 
 function Item:Update()
 	self.info = self:GetInfo()
-	self.hasItem = self.info.id and true -- for blizzard template
-	self.readable = self.info.readable -- for blizzard template
+	self.hasItem = false -- for blizzard template
+	self.readable = false  -- for blizzard template
+	self.info.count = 1  -- for blizzard template
```

Comment:
This is overwriting already set count value breaking the display of item stack size